backend/core/src/user-preferences/entities/user-preferences.entity.ts

```diff
@@ -17,4 +17,8 @@ export class UserPreferences {
   @Column("boolean", { default: false })
   @Expose()
   sendSmsNotifications?: boolean
+
+  @Column("jsonb")
+  @Expose()
+  favorites: Array<string>
```

Comment:
It might be better to define this as a OneToMany mapping to Listing[]. That way, all the fancy backend SQL query stuff will work out of the box if we want to go further with this (like only show favorites that match a particular criteria).

Reply:
Gabe made a good point in my DMs in that this should probably be a ManyToMany relationship, since Listings can have many Users who've favorited it.